src/Pages/Withdrawals.razor

```diff
@@ -77,6 +77,9 @@
                                      && LoggedUser?.Id != context.UserRequestorId)
                             {
                                 <Button Color="Color.Danger" Clicked="()=> ShowRejectCancelModal(context, WalletWithdrawalRequestStatus.Rejected)">Reject</Button>
+                            } else if (LoggedUser?.Id == context.UserRequestorId)
+                            {
+                                <Button Color="Color.Danger" Clicked="()=> ShowRejectCancelModal(context, WalletWithdrawalRequestStatus.Cancelled)">Cancel</Button>
```

Comment:
You really don't need this 3rd clause, it will never run.

I think you can even simplify it to
```
@if (LoggedUser?.Id == context.UserRequestorId)
{
    <Button Color="Color.Danger" Clicked="()=> ShowRejectCancelModal(context, WalletWithdrawalRequestStatus.Cancelled)">Cancel</Button>
}
else 
{
    <Button Color="Color.Danger" Clicked="()=> ShowRejectCancelModal(context, WalletWithdrawalRequestStatus.Rejected)">Reject</Button>
}
```